es-bs-base/scss/vue/components/_pagination-nav.scss

```diff
@@ -0,0 +1,2 @@
+// <pagination-nav> shares <pagination> SCSS
+@forward "pagination";
```

Comment:
What is this doing? Feels like we would only need to forward if we are changing the definition.

Reply:
It makes it such that `@use 'pagination-nav'` would resolve to loading all of `_pagination.scss` downstream.

This is what it looks like pre-module SCSS https://github.com/bootstrap-vue/bootstrap-vue/blob/dev/src/components/pagination-nav/index.scss